### PR TITLE
Small warning for windows users

### PR DESCRIPTION
**Pull request by gkaykck**
I've dropped a small comment for windows users for `clever ssh` command

### CODE REVIEW

File: commons/clever-tools/ssh-access.md [hunk 1]
```diff
@@ -45,6 +45,9 @@ If multiple instances are running, you will be asked which one to use:
     > Your choice: 1
     > Opening an ssh shell
     > bas@11281f38-31ff-43a7-8595-a2d82630c32b ~ $
+    
+###For Windows users
```
Comment (BlackYoup) [inline commons/clever-tools/ssh-access.md]: Maybe a space after the `###` ? That's just cosmetic though

File: commons/clever-tools/ssh-access.md [hunk 2]
```diff
@@ -45,6 +45,9 @@ If multiple instances are running, you will be asked which one to use:
...
+`clever ssh` command will fail on PowerShell or cmd.exe if there is no ssh.exe in your path, which is true for most users. To be able to overcome this problem, the most straightforward solution is, starting `clever ssh` command from `git-bash`, which should be already installed on your system.
```
Comment (BlackYoup) [inline commons/clever-tools/ssh-access.md]: I would get rid of one of the comma at the end and maybe a bit of wording: `the most straightforward solution is to start "clever ssh" from "git-bash"`

File: commons/clever-tools/ssh-access.md (hunk 1 above)
Comment (cnivolle) [reply commons/clever-tools/ssh-access.md]: I approve that. Let's roll it this way.

File: commons/clever-tools/ssh-access.md (hunk 2 above)
Comment (cnivolle) [reply commons/clever-tools/ssh-access.md]: Done ✅

**Pull request merged**